lms/static/js/logout.js

```diff
@@ -11,13 +11,16 @@
         var $iframeContainer = $('#iframeContainer'),
             $iframes = $iframeContainer.find('iframe'),
             redirectUrl = $iframeContainer.data('redirect-url');
+        var isSamlRequired = document.getElementsByClassName("saml")[0] ? true : false
```

Comment:
Use jquery to figure out value of `isSamlRequired` and filtering on classname does not make sense in case we are looking for single element. Identifier lookup is needed here.

Reply:
done